src/components/Header/ProfileBtn.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import ProfileImg from '../../asset/images/profile.jpg';
+
+const ProfileBtnWrap = styled.button`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 105px;
+    height: 32px;
+    padding: 2px;
+    border-radius: 20px;
+    position: absolute;
+    top: 50%;
+    right: 17px;
+    transform: translateY(-50%);
+    background-color: #080810;
+    cursor: pointer;
+
+    &:hover {
+        background-color: #393939;
+    }
+`;
+
+const ProfileBtnImg = styled.div`
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+    background: url(${ProfileImg}) top left no-repeat;
+    background-size: cover;
+`;
+
+const ProfileName = styled.div`
+    color: #fff;
+    font-size: 0.75rem;
+    font-weight: 900;
+`;
+
+const ProfileArrow = styled.div`
+    width: 20px;
+    height: 10px;
+    margin-top: 8px;
+    position: relative;
+    overflow: hidden;
+    &::after {
+        content: '';
+        display: block;
+        width: 7px;
+        height: 7px;
+        position: absolute;
+        top: -30%;
+        left: 50%;
+        background-color: #fff;
+        transform: translateX(-50%) rotate(45deg);
+    }
+`;
+
+interface IProps {
```

Comment:
정말 죄송합니다. 이부분 동건님께 알려드리고 찾아봤는데 I쓰지말라는 문서가 있더라고요...
참고하세요ㅜㅜ(어디는 붙이라고하기도해서 이건 선택하시면될거같습니다)
https://dirask.com/posts/TypeScript-interface-naming-convention-x1RwqD

Reply:
```suggestion
interface ProfileBtnProps {
```
Props이름을 사용하게된다면 Props으로만 이름짓는 것이아니라 앞에 컴포넌트 이름을 붙여주는게 더 좋아요

Reply:
아하 컴포넌트명을 붙이는군요.. type이나 interface에대한 네이밍이 고민이었는데 감사합니다